src/main/resources/iudx/catalogue/server/validator/providerItemSchema.json

```diff
@@ -1,310 +1,319 @@
 {
```

Comment:
What is the change here?
Formatting only?

Reply:
added **maxLength:512** to check the length of providerId

Reply:
Ok. It has increased the diff because of some indentation mismatch. I will fix it.